src/mcnp/mcnp5/CMakeLists.txt

```diff
@@ -52,7 +52,6 @@ set(CMAKE_Fortran_FLAGS_RELWITHDEBINFO "-O1 -g")
 
 # C compiler flags
 if (CMAKE_C_COMPILER_ID STREQUAL "Intel")
-  #set(CMAKE_C_FLAGS "${CMAKE_Fortran_FLAGS} -mcmodel=medium")
```

Comment:
The reason these commented lines are here is that they appear in the makefiles for native MCNP, but I needed to turn them off. I don't remember why.

Reply:
Should they be left in and out in use? Or is it okay to remove these lines?

Reply:
Ehhhh yeah let's just delete them